### ISSUE TITLE
Rspec tests fail on views with erb comments with single quotes

### ISSUE THREAD

**Issue by djfpaagman** [glebm/i18n-tasks#429]
I found that 1.0.x fails for me, and narrowed it down to erb comments that have single quotes in them, for example when used as a apostrophe in a sentence.

This is the simpelest test case I can make that reproduces the error:

```erb 
<%# this will fail: ' %>
```

```
  1) I18n does not have missing keys
     Failure/Error: raise Parser::SyntaxError, diagnostic

     I18n::Tasks::CommandError:
       Error scanning app/views/test.html.erb: unterminated string meets end of file
     # (eval):3:in `_racc_do_parse_c'
     # (eval):3:in `do_parse'
     # ------------------
     # --- Caused by: ---
     # Parser::SyntaxError:
     #   unterminated string meets end of file
     #   (eval):3:in `_racc_do_parse_c'

  2) I18n does not have unused keys
     Failure/Error: raise Parser::SyntaxError, diagnostic

     I18n::Tasks::CommandError:
       Error scanning app/views/test.html.erb: unterminated string meets end of file
     # (eval):3:in `_racc_do_parse_c'
     # (eval):3:in `do_parse'
     # ------------------
     # --- Caused by: ---
     # Parser::SyntaxError:
     #   unterminated string meets end of file
     #   (eval):3:in `_racc_do_parse_c'
```

The rake tasks do work properly.

**Comment by davidwessman** [comment]
Might have time to look at it today. 

@glebm would it make sense to rescue in the parser and skip that node if it fails? E.g. if there is "wrong syntax" it will miss translations on that row as well? Should we print an error message?

**Comment by glebm** [comment]
@davidwessman But why is it parsing a comment?

**Comment by glebm** [comment]
Putting a space between a `%` and `#` makes it work:

```
<% # this should not fail: ' %>
```

Looks like a bug in the parser somewhere.

Without a space we get this in handler missing for `node.inspect`:

```
s(:erb,\n  s(:indicator, \"#\"), nil,\n  s(:code, \" this should not fail: ' \"), nil)
```

With a space we get:

```
s(:erb, nil, nil,
  s(:code, " # this should not fail: ' "), nil)
```

**Comment by davidwessman** [comment]
Ah, that seems to be a limit in better-html :/ I guess we could handle that AST-node for now

**Comment by glebm** [comment]
@davidwessman I've given it a go at #430

**Comment by glebm** [comment]
Released v1.0.4

**Comment by djfpaagman** [comment]
Thanks @davidwessman and @glebm for the super fast follow up :+1:
